storages/backends/s3boto3.py

```diff
@@ -518,8 +531,13 @@ def _save_content(self, obj, content, parameters):
         if self.default_acl:
             put_parameters['ACL'] = self.default_acl
         content.seek(0, os.SEEK_SET)
```

Comment:
It seems that `content_autoclose` should be used in this and the next line instead of `content`.

Reply:
Feel free to make changes here